shared/constants/network.ts

```diff
@@ -586,8 +584,6 @@ export const CHAIN_ID_TO_CURRENCY_SYMBOL_MAP = {
     CHAINLIST_CURRENCY_SYMBOLS_MAP.BLACKFORT_EXCHANGE_NETWORK,
   [CHAINLIST_CHAIN_IDS_MAP.CONFLUX_ESPACE]:
     CHAINLIST_CURRENCY_SYMBOLS_MAP.CONFLUX_ESPACE,
-  [CHAINLIST_CHAIN_IDS_MAP.FUSE_MAINNET]:
-    CHAINLIST_CURRENCY_SYMBOLS_MAP.FUSE_MAINNET,
```

Comment:
These keys were duplicates, and don't pass the type checker in v5.3.3.

Reply:
I just picked one. I'll make a new PR to switch it back is that's okay with you. This PR requires lots of different teams to review, and I don't want to have to track everyone down again if I don't have to.